socialdistribution_root/apis/authors/views.py

```diff
@@ -108,17 +108,17 @@ def get(self, request: HttpRequest):
 
 
 
-def getAuthor(author_id: str) -> User:
+def getAuthor(author_id: str) -> Author:
     try:
-        author = User.objects.get(id=author_id)
+        author = User.objects.get(pk=author_id)
```

Comment:
This should most likely be Author.objects.get... because User model only has auth stuff

Reply:
fixed you are correct